src/components/BackButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+
+const StyledBackButtonWrapper = styled.div`
+  width: 100%;
+`;
+
+const BackButton = () => {
+  const location = useLocation();
+  const { t } = useTranslation();
+  const backHref = (location.pathname.includes('/editresource') ? '/resources/user/current' : '/') + location.search;
+
+  return (
+    <>
+      {location.pathname === '/' || location.pathname.includes('/content/main') ? (
+        <></>
+      ) : (
+        <>
+          <StyledBackButtonWrapper>
+            <Button data-testid="navigation-back-button" href={backHref} color="primary" variant="outlined">
```

Comment:
skulle vi ikke gjort om knappen til en tekst-lenke isteden? Det ville vært både mer naturlig for navigering, og det vil ta mindre plass ?